GoogleDataTransport/GDTCORLibrary/GDTCORFlatFileStorage.m

```diff
@@ -227,7 +245,7 @@ - (void)storeEvent:(GDTCOREvent *)event
     // Write the transport bytes to disk, get a filename.
     GDTCORAssert([event.dataObject transportBytes],
                  @"The event should have been serialized to bytes");
-    NSError *error = nil;
+    error = nil;
```

Comment:
The nil initialization is only necessary for clarity - but if done, it should be at the declaration above - https://stackoverflow.com/a/10022295/556617

Reply:
This is to clear the error object from above if it's non-nil for some reason.